client/src/components/TextForm.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { Header, Segment, Container, Button, Form } from "semantic-ui-react";
+import languageOptions from "./LanguageOptions";
+
+class TextForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      fromLanguage: "",
+      toLanguage: "",
+      haveAllFielsValue: false,
+      dueDate: "",
+      text: ""
+    };
+  }
+
+  handleSubmit = () => {
+    console.log(this.state);
+    this.setState({
+      fromLanguage: "",
+      toLanguage: "",
+      dueDate: "",
+      text: "",
+      haveAllFielsValue: false
+    });
+    console.log(this.state);
+  };
+
+  checkAllFields = () => {
+    let haveAllFielsValue = false;
+    if (this.state.fromLanguage && this.state.toLanguage && this.state.text) {
+      haveAllFielsValue = true;
+    }
+    this.setState({ haveAllFielsValue: haveAllFielsValue });
+  };
+
+  handleChange = (e, { value, name }) => {
+    this.setState({ [name]: value }, () => this.checkAllFields());
+  };
+
+  render() {
+    const {
+      fromLanguage,
+      toLanguage,
+      dueDate,
+      text,
+      haveAllFielsValue
+    } = this.state;
+
+    return (
+      <Container>
+        <Header as="h2">
+          To get your text translated, you need to fill in the form
```

Comment:
The main title of the page can be "Add document" and the description below can be what you wrote here.

Reply:
Changed